tests/ert3/storage/test_storage.py

```diff
@@ -314,3 +322,33 @@ def test_delete_experiment(tmpdir, ert_storage):
     ert3.storage.delete_experiment(workspace=tmpdir, experiment_name="test")
 
     assert "test" not in ert3.storage.get_experiment_names(workspace=tmpdir)
+
+
+@pytest.mark.requires_ert_storage
+def test_get_ensemble_responses(tmpdir, ert_storage):
+    ert3.storage.init(workspace=tmpdir)
+    responses = ["resp1", "resp2"]
+    records = responses + ["rec1"]
+    experiment = "exp"
+    ert3.storage.init_experiment(
+        workspace=tmpdir,
+        experiment_name=experiment,
+        parameters=[],
+        ensemble_size=1,
+        responses=responses,
+    )
+    for name in records:
+        ensemble_record = ert3.data.EnsembleRecord(
+            records=[ert3.data.Record(data=[1, 2, 3])]
+        )
+        ert3.storage.add_ensemble_record(
+            workspace=tmpdir,
+            record_name=name,
+            ensemble_record=ensemble_record,
+            experiment_name=experiment,
+        )
+
+    fetched_ensemble_responses = ert3.storage.get_experiment_responses(
+        workspace=tmpdir, experiment_name=experiment
+    )
+    assert "rec1" not in fetched_ensemble_responses
```

Comment:
Perhaps also assert the expected content of `fetched_ensemble_responses`?

What is the behavior if a record name is both a parameter and a response? It would be good with a few more tests for the unhappy path here!

Reply:
Added a bit more tests: 
- record name both in parameter and responses
- fetching responses when response-records are not loaded
- fetching responses when responses are not defined